addons/scope_off/fn_init.sqf

```diff
@@ -0,0 +1,30 @@
+[
+    "cnto_scope_remover_enable",
+    "CHECKBOX",
+    ["Enable", "Remove any scopes of more than 1x from AI (that are NOT Snipers or Marksman) units upon spawning."],
+    ["CNTO Additions", "Scope remover"],
+    true,  /* default */
+    true,  /* isGlobal */
+    nil,   /* script */
+    true   /* needRestart */
+] call CBA_settings_fnc_init;
+
+
+0 = [] spawn {
```

Comment:
Wrap the spawn in a CBA event like https://github.com/freghar/arma-additions/blob/master/addons/marker_enh/fn_init.sqf - you must not check for `cnto_scope_remover_enable` before CBA settings are initialized. Also, since you're now using CBA events, add `cba_events` to `requiredAddons[]`.